### ISSUE TITLE
Accessibility - Sign In Screen - VoiceOver Does Not Announce DOL Sign In SCreen Information

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#8]
Once I enter my e-mail address it takes me to the DOL styled login and then never says anything like what page the user is on etc. I would expect the cursor to focus on the e-mail address or the password field instead of the cancel button at the top of the page. The user should at least be told what page they are on.

**Comment by jfc3-dol** [comment]
Well, that could be an issue for users that use assistive technology like a screen reader. We might want to point issues like this out the users.

Please let me close the issues that way I can keep track of what has been fixed and reviewed.
